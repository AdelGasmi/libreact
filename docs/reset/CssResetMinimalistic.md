# `<CssResetMinimalistic>`

## Usage

Simply render it in your React app

```jsx
import CssResetMinimalistic from 'mol-fe-react/lib/reset/CssResetMinimalistic';

<CssResetMinimalistic />
```

## Contents

```js
{
  '*': {
      pad: 0,
      mar: 0,
  },
}
```
