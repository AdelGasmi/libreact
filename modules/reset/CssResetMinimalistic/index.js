import createCssResetComponent from '../createCssResetComponent';
export const css = {
    '*': {
        pad: 0,
        mar: 0,
    },
};
const CssResetMinimalistic = createCssResetComponent(css);
export default CssResetMinimalistic;
