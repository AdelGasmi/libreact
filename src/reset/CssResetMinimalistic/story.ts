import {createElement as h} from 'react';
import {storiesOf} from '@storybook/react';
import {action} from '@storybook/addon-actions';
import {linkTo} from '@storybook/addon-links';
import CssResetMinimalistic from '.';
import ShowDocs from '../../../.storybook/ShowDocs'

storiesOf('CSS resets/CssResetMinimalistic', module)
  .add('Documentation', () => h(ShowDocs, {md: require('../../../docs/en/reset/CssResetMinimalistic.md')}))
  .add('Example', () =>
    h(CssResetMinimalistic)
  );
