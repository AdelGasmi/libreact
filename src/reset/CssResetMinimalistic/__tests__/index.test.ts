import {createElement as h} from 'react';
import {renderToStaticMarkup} from 'react-dom/server';
import CssResetMinimalistic from '..';

describe('<CssResetMinimalistic>', () => {
  it('renders as expected', () => {
    const html = renderToStaticMarkup(h(CssResetMinimalistic));

    expect(html).toMatchSnapshot();
  });
});
