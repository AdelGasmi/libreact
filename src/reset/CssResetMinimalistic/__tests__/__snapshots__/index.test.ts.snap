// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CssResetMinimalistic> renders as expected 1`] = `
"<style>* {
    padding: 0;
    margin: 0;
}
</style>"
`;
